src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,32 +2,35 @@
 
 public class DefaultCountingOutRhymer {
 
-	private int[] numbers = new int[12];
+	private static final int const_11 = 11;
```

Comment:
nazwy zmiennych jednak lepiej pisać camelCase, znacznie lepiej to wygląda i jest łatwiejsze w odbiorze.
Dodatkowo nazwy zmiennych powinny opisywać czym są, co można powiedzieć, że czytając nazwe zmiennej np. isHidden, to wiem, że przechowuje w sobie wartość booleana (przynajmniej powinien, wg standardu). 

Zmienne const_11, const_12 itd, nie mówią nic o tym, czym one są.  

Warto też nie używać słów jakby  "niedozwolonych" w nazwach zmiennych, o  ile w Javie samo const nie jest takie wrażliwe, to np. w JS-ie już mając zapis: const const_11 = 11; brzmi nazbyt dziko i nie przekazuje żadnej informacji